chroma_core/services/job_scheduler/job_scheduler.py

```diff
@@ -1106,13 +1106,10 @@ def key(td):
         # if we have an entry with 'root'=true then move it to the front of the list before returning the result
         return sorted(sorted_list, key=lambda entry: entry.get("root", False), reverse=True)
 
-    def create_client_mount(self, host_id, filesystem_id, mountpoint):
+    def create_client_mount(self, host_id, filesystem, mountpoint):
```

Comment:
pass fs name instead of the whole filesystem object

Reply:
Actually, I believe this is the name, but I will double check.

Reply:
I mean we cannot pass the whole filesystem model into this RPC as it gets copied and is subject to mutation errors.

Previously it was `filesystem_id` and was looked up inside this method, now it should be `filesystem_name` and still be looked up inside this method.

Reply:
I wonder if we should keep it as the id.

Reply:
It's probably better as name so you don't need to look it up outside the RPC.